site/examples/navigation.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const menuNavigation = `<nav'>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3 is-active' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-home'/></svg> Home
```

Comment:
The vertical centering is better if you use flexbox. Or you could wrap the text in a `<div class='inline-block align-middle'>`